backend/editor/models.py

```diff
@@ -16,3 +16,70 @@ class Header(Marginal):
 
 class Footer(Marginal):
     pass
+
+
+# Models for FastAPI
+
+
+class ImportFromGithubParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+
+
+class ImportFromGithubResponse(BaseModel):
+    status: str
+
+
+class CreateNodeParameters(BaseModel):
+    branch: str
+    taxonomy_name: str

```

Comment:
Those parameters (branch, taxonomy_name) are repeated in many cases, we have to factor them out in a class and inherit it in other classes.

This make sense combine with the fact that we must provide descriptions.